_posts/2021-04-23-status-update.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: post
+title: "2021-04-23: IAC & DevX Team Status Update"
+author: da-ar
+categories:
+  - team
+  - status
+  - puppet-camp
+  - litmus
+  - bolt
+  - facter
+  - cloud-ci
+  - rsapi
+  - devx
+  - dsc
+tags:
```

Comment:
Could do with filling out the tags - I'd nearly say what's in categories should be in `tags` and keep `categories`: `status`, `team`